content/ch-states/atoms-computation.ipynb

```diff
@@ -199,19 +199,16 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "n = 8\n",
-    "n_q = n\n",
-    "n_b = n\n",
-    "qc_output = QuantumCircuit(n_q,n_b)"
+    "qc_output = QuantumCircuit(8)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "This circuit, which we have called `qc_output`, is created by Qiskit using `QuantumCircuit`. The number `n_q` defines the number of qubits in the circuit. With `n_b` we define the number of output bits we will extract from the circuit at the end.\n",
+    "This circuit, which we have called `qc_output`, is created by Qiskit using `QuantumCircuit`.The `QuantumCircuit` takes numbers of Qubits to be used in Quantum Circuit as a argument. \n",
     "\n",
-    "The extraction of outputs in a quantum circuit is done using an operation called `measure`. Each measurement tells a specific qubit to give an output to a specific output bit. The following code adds a `measure` operation to each of our eight qubits. The qubits and bits are both labelled by the numbers from 0 to 7 (because that’s how programmers like to do things). The command `qc_output.measure(j,j)` adds a measurement to our circuit `qc_output` that tells qubit `j` to write an output to bit `j`."
+    "The extraction of outputs in a quantum circuit is done using an operation called `measure_all()`. Each measurement tells a specific qubit to give an output to a specific output bit. The following predefined function adds a `measure` operation to each of our eight qubits.  The command `qc_output.measure_all()` adds a measurement to our circuit `qc_output` that measure all qubits at a instance."
```

Comment:
```suggestion
    "The extraction of outputs in a quantum circuit is done using an operation called `measure_all()`. Each measurement tells a specific qubit to give an output to a specific output bit. The command `qc_output.measure_all()` adds a measurement to each qubit in the circuit `qc_output`, and also adds some classical bits to write the output to."
```
I think "The following code..." referred to the loop in the cell below, so can probably remove. Also think we should mention `measure_all()` adds some cbits too, as `.measure()` doesn't do this so might trip people up.

Reply:
Yeah `measure_all()` adds extra classical bits. It's good idea to mention that to avoid any confusion.